allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -18,37 +18,51 @@
 LONG_STIM_THRESHOLD = 0.2     # seconds
 MAX_MONITOR_DELAY = 0.07     # seconds
 
-VERSION_1_KEYS = {
-    "photodiode": "stim_photodiode",
-    "2p": "2p_vsync",
-    "stimulus": "stim_vsync",
-    "eye_camera": "cam2_exposure",
-    "behavior_camera": "cam1_exposure",
-    "acquiring": "2p_acquiring",
-    "lick_sensor": "lick_1"
-    }
-
-# MPE is changing keys. This isn't versioned in the file.
-VERSION_2_KEYS = {
-    "photodiode": "photodiode",
-    "2p": "2p_vsync",
-    "stimulus": "stim_vsync",
-    "eye_camera": "eye_tracking",
-    "behavior_camera": "behavior_monitoring",
-    "acquiring": "2p_acquiring",
-    "lick_sensor": "lick_sensor"
-    }
-
-
-def get_keys(sync_dset):
-    """Get the correct lookup for line labels.
-
-    This method is fragile, but not all old data contains the full list
-    of keys.
+
+def get_keys(sync_dset: Dataset) -> dict:
+    """
+    Gets the correct keys for the sync file by searching the sync file
```

Comment:
Does this docstring still describe what it is you want the function to do? In particular, is it defaulting to `use old keys if none of the correct ones are found`? What it looks like is that if a key is not found, it removes that key from the returned dict. If I'm reading it right, either the docstring needs to change to describe the function, or the function needs to change to perform what the docstring says.

Reply:
Thanks for pointing this out.